crates/bitwarden-core/src/auth/auth_client.rs

```diff
@@ -185,9 +185,12 @@ pub enum TrustDeviceError {
 
 #[cfg(feature = "internal")]
 fn trust_device(client: &Client) -> Result<TrustDeviceResponse, TrustDeviceError> {
-    let enc = client.internal.get_encryption_settings()?;
+    use crate::key_management::SymmetricKeyId;
 
-    let user_key = enc.get_key(&None)?;
+    let key_store = client.internal.get_key_store();
+    let ctx = key_store.context();
+    #[allow(deprecated)]
+    let user_key = ctx.dangerous_get_symmetric_key(SymmetricKeyId::User)?;
```

Comment:
Could we comment all instances of deprecated why we do this, and if we plan on eventually replacing it add a link to a ticket?

Reply:
Added ticket references and short explanations to all the non-test uses, is this what you had in mind?
https://github.com/bitwarden/sdk-internal/pull/8/commits/36e8a51b05d26c536a66beb9d154e7afc2188ac0